S/Spot_julia/build_tarballs.jl

```diff
@@ -0,0 +1,102 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Spot_julia"
+version = v"2.9.7"
+julia_version = v"1.6.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("http://www.lrde.epita.fr/dload/spot/spot-2.9.7.tar.gz","1eea67e3446cdbbbb705ee6e26fd869020cdb7d82c563fead9cb4394b9baa04c"),
+    GitSource("https://github.com/MaximeBouton/spot_julia.git", "b68e1ce02373725f8fd4edbc085b341496c920ab")
+    ]
+    
+    # Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/spot-2.9.7/
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --disable-python
+make -j${nproc}
+make install
+
+# build with cmake 
+cd $WORKSPACE/srcdir/spot_julia/spot_julia
+
+
+SPOT_LIB_DIR=${libdir}
+SPOT_INCLUDE_DIR=${includedir}
+if [[ $target == *"mingw"* ]]; then
+    SPOT_LIB_DIR=${bindir}
+fi
```

Comment:
> It should be fixed in the latest commit

Is it?  I still see these useless lines

```suggestion
```

Reply:
I believe this is not useless, otherwise cmake is going to link with the `.dll.a` instead of the `.dll` on windows. 
The `.dll`s are placed in `bindir` by libtool.

Reply:
oh my bad, I missed your other comment saying that `.dll.a` are  import library. I am a novice with this so I don't fully grasp the implications yet. Anyway, I will remove this part.

Reply:
> I believe this is not useless

It _is_ useless because `${libdir}` is `${prefix}/lib` on Unix systems and `${prefix}/bin` on Windows, as explained in the [documentation](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables), and the reason why I always insist on using the most specific environment variable